src/app/public/modules/text-editor/text-editor.component.ts

```diff
@@ -105,10 +115,33 @@ export class SkyTextEditorComponent implements AfterViewInit, ControlValueAccess
   @Input()
   public autofocus: boolean = false;
 
+  /**
+   * Indicates whether to disable the text editor.
+   * @default false
+   */
+  @Input()
+  public set disabled(value: boolean) {
+    const coercedValue = SkyFormsUtility.coerceBooleanProperty(value);
+    if (coercedValue !== this.disabled) {
+      this._disabled = coercedValue;
+      this.changeDetector.markForCheck();
+      this.adapterService.toggleEditorAbility(this.id, this._disabled, this.focusableChildren);
```

Comment:
Maybe not necessary, but should we move the `.markForCheck()` after the toggle so that we are sure to capture all attributes/classes/etc... getting updated before we ask Angular to update the view?

Reply:
fixed in commit: ["mark for check after toggle"](https://github.com/blackbaud/skyux-text-editor/pull/21/commits/7087d27d3adc61fafa6adcd2f2cbeca69e5906f1)